.gitignore

```diff
@@ -108,8 +108,6 @@ ipch/
 *.vspx
 *.sap
 
-# Visual Studio Trace Files
-*.e2e
```

Comment:
Does this need to be removed?

Reply:
Yes, so if you are not removing this line from the.gitignore and you are changing any test solution file and trying to commit it, that file is not showing up in the commit list.

Reply:
Do we need to commit any `*.e2e` files to the repository?

Reply:
Ohhhh! I get it. This makes the project file not want to commit.